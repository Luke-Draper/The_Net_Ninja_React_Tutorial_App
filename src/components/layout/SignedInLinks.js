import React, { Component } from "react";

class SignedInLinks extends Component {
	render() {
		return (
			<nav className="signed-in-links">
				<h2>SignedInLinks</h2>
			</nav>
		);
	}
}

export default SignedInLinks;
