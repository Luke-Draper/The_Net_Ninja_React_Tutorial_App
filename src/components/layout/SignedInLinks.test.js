import React from "react";
import SignedInLinks from "./SignedInLinks";
import { MemoryRouter } from "react-router-dom"; //Required when component uses a Route or Link
import renderer from "react-test-renderer";

test("SignedInLinks renders without crashing", () => {
	const component = renderer.create(
		<MemoryRouter initialEntries={["/"]}>
			<SignedInLinks />
		</MemoryRouter>
	);
	expect(component.toJSON()).toMatchSnapshot();
});
