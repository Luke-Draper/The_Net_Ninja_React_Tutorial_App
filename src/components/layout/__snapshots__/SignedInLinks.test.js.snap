// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignedInLinks renders without crashing 1`] = `
<nav
  className="signed-in-links"
>
  <h2>
    SignedInLinks
  </h2>
</nav>
`;
